lib/jwtverify.lua

```diff
@@ -25,6 +25,7 @@ if not config then
   config = {
```

Comment:
Could you also copy the updated `lib/jwtverify.lua` to `example/haproxy/lib/jwtverify.lua`? It's the same file, but copied into the example.